src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java

```diff
@@ -1,2703 +1,2708 @@
+                    playerData.setVisibleBoundaries(null);
+                    return;
+                }
+
+                Claim claim = this.dataStore.getClaimAt(clickedBlock.getLocation(), false /*ignore height*/, playerData.lastClaim);
+
+                //no claim case
+                if (claim == null)
+                {
+                    // alert plugins of a claim inspection, return if cancelled
+                    ClaimInspectionEvent inspectionEvent = new ClaimInspectionEvent(player, clickedBlock, null);
+                    Bukkit.getPluginManager().callEvent(inspectionEvent);
+                    if (inspectionEvent.isCancelled()) return;
+
+                    GriefPrevention.sendMessage(player, TextMode.Info, Messages.BlockNotClaimed);
+
+                    playerData.setVisibleBoundaries(null);
+                }
+
+                //claim case
+                else
+                {
+                    // alert plugins of a claim inspection, return if cancelled
+                    ClaimInspectionEvent inspectionEvent = new ClaimInspectionEvent(player, clickedBlock, claim);
+                    Bukkit.getPluginManager().callEvent(inspectionEvent);
+                    if (inspectionEvent.isCancelled()) return;
+
+                    playerData.lastClaim = claim;
+                    GriefPrevention.sendMessage(player, TextMode.Info, Messages.BlockClaimed, claim.getOwnerName());
+
+                    //visualize boundary
+                    BoundaryVisualization.visualizeClaim(player, claim, VisualizationType.CLAIM);
+
+                    if (player.hasPermission("griefprevention.seeclaimsize"))
+                    {
+                        GriefPrevention.sendMessage(player, TextMode.Info, "  " + claim.getWidth() + "x" + claim.getHeight() + "=" + claim.getArea());
+                    }
+
+                    //if permission, tell about the player's offline time
+                    if (!claim.isAdminClaim() && (player.hasPermission("griefprevention.deleteclaims") || player.hasPermission("griefprevention.seeinactivity")))
+                    {
+                        if (claim.parent != null)
+                        {
+                            claim = claim.parent;
+                        }
+                        Date lastLogin = new Date(Bukkit.getOfflinePlayer(claim.ownerID).getLastPlayed());
+                        Date now = new Date();
+                        long timeElapsed = now.getTime() - lastLogin.getTime();
+                        long daysElapsed = timeElapsed / (1000 * 60 * 60 * 24);
+                        long hoursElapsed = (timeElapsed / (1000 * 60 * 60)) % 24;
+                        long minutesElapsed = (timeElapsed / (1000 * 60)) % 60;
+                        long secondsElapsed = (timeElapsed / 1000) % 60;
+
+                        String formattedTime = String.format("%d dias, %d horas, %d minutos e %d segundos", daysElapsed, hoursElapsed, minutesElapsed, secondsElapsed);

```

Comment:
GP's default should be English. You would be better-served by having the more granular measurements added as additional parameters.